components/renku_data_services/notebooks/api/classes/k8s_client.py

```diff
@@ -92,135 +135,138 @@ async def get_pod_logs(self, name: str, max_log_lines: Optional[int] = None) ->
                 # NOTE: This occurs when the container is still starting but we try to read its logs
                 continue
             except NotFoundError:
-                raise errors.MissingResourceError(message=f"The session pod {name} does not exist.")
+                raise errors.MissingResourceError(message=f"The session pod {pod_name} does not exist.")
             except ServerError as err:
                 if err.status == 404:
-                    raise errors.MissingResourceError(message=f"The session pod {name} does not exist.")
+                    raise errors.MissingResourceError(message=f"The session pod {pod_name} does not exist.")
                 raise
             else:
                 logs[container] = "\n".join(clogs)
         return logs
 
-    async def get_secret(self, name: str) -> Secret | None:
-        """Read a specific secret from the cluster."""
-        try:
-            secret = await Secret.get(name, self.namespace)
-        except NotFoundError:
+    async def _get_secret(self, name: str) -> Secret | None:
```

Comment:
This is not really used anywhere as far as I can tell. I even tried to look for it with the old name. So we can just remove it fully.